network-config-analyzer/NetworkPolicy.py

```diff
@@ -101,6 +101,9 @@ def get_policy_type(policy):
             return NetworkPolicy.PolicyType.Unknown
 
         policy_type = NetworkPolicy.PolicyType.Unknown
+        if not isinstance(kind, str):
+            return policy_type
```

Comment:
replace lines 103-105 with the following code to be more consistent with code at lines 95-101

if not isinstance(kind, str):
   return NetworkPolicy.PolicyType.Unknown

policy_type = NetworkPolicy.PolicyType.Unknow